src/Application.php

```diff
@@ -28,6 +28,9 @@ public function __construct($name, $version)
             ->addOption(
                 new InputOption('--define', '-D', InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY, 'Define a configuration item value.', [])
             );
+
+        $selfUpdateCommand = new SelfUpdateCommand('self:update');
+        $this->add($selfUpdateCommand);
```

Comment:
This doesn't do the right thing unless running as a phar; perhaps phar mode should be detected, and this command should not be added when not running as a phar.